plugins/woocommerce/src/Admin/API/OnboardingPlugins.php

```diff
@@ -155,7 +156,12 @@ public function install_and_activate_async( WP_REST_Request $request ) {
 		$plugins = $request->get_param( 'plugins' );
 		$job_id  = uniqid();
 
-		WC()->queue()->add( 'woocommerce_plugins_install_and_activate_async_callback', array( $plugins, $job_id ) );
+		WC()->queue()->add(
+			'woocommerce_plugins_install_and_activate_async_callback',
+			array( $plugins, $job_id ),
+			'',
+			ActionQueuePriority::URGENT
```

Comment:
`ActionQueuePriority::URGENT` - toying around, not sure what this action is supposed to do.